### PR TITLE
58-editComputerForm - Edit a Computer

### PR DESCRIPTION
**Pull request by matthamil**
## Status
**READY**

## Documentation

[X] I have fully documented features in this PR's code. Please put an `X` in the box to confirm.

[ ] This PR does **NOT** require documentation.

## Description
A few sentences describing the overall goals of the pull request's commits.

## Resolves Issue Number

https://github.com/TeamCharles/workforce-management/issues/58

## Deploy Notes

```sh
git pull --prune
git checkout 58-editComputerForm
dotnet run
```

## Steps to Test or Reproduce

1. Navigate to `localhost:5000`

1. Click the "Computers" button in the navigation

1. Click the "Edit" button on a Computer

1. Attempt to edit a computer

1. Edit the computer successfully

1. See the updated info in `localhost:5000/Computer`

## Impacted Files in Application

* Views/Computer/Index.cshtml

* Views/Computer/Edit.cshtml

* ViewModels/Computer/ComputerEdit.cs

* Controllers/ComputerController.cs

### CODE REVIEW

**Comment by matthamil** [comment]
Ready for review @TeamCharles/team

File: src/workforce-management/Views/Computer/Edit.cshtml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+                <input asp-for="ComputerToEdit.Model" class="form-control computerAddModel" />
+                <span asp-validation-for="ComputerToEdit.Model" class="text-danger"></span>
+            </div>
+        </div>
+
+        @Html.Hidden("ComputerToEdit.ComputerId", Model.ComputerToEdit.ComputerId);
+
+        <div class="form-group">
+            <div class="col-md-offset-2 col-md-10">
+                <input type="submit" value="Save Changes" class="btn btn-default" id="computerAddBtn" disabled="true" />
```
Comment (MattKraatz) [inline src/workforce-management/Views/Computer/Edit.cshtml]: should `Save Changes` shouldn't be disabled on load? I guess it doesn't make sense for a user to try and submit the form after making no changes, but it seems like undesirable behavior to require a change before enabling that button.

File: src/workforce-management/Views/Computer/Index.cshtml [hunk 2]
```diff
@@ -26,6 +26,7 @@
             <th>Make</th>
             <th>Model</th>
             <th>Employee</th>
+            <th>@* Empty cell where the edit button will go in each row *@</th>
```
Comment (MattKraatz) [inline src/workforce-management/Views/Computer/Index.cshtml:29]: nice comments

**Review by MattKraatz — APPROVED** [review]
Great work 👍 :

**Comment by garrett-vangilder** [comment]
LD(ope)TM

File: src/workforce-management/Views/Computer/Edit.cshtml (hunk 1 above)
Comment (matthamil) [reply src/workforce-management/Views/Computer/Edit.cshtml]: Done

**Comment by daynewright** [comment]
LGTM!

**Pull request merged**